### DISCUSSION TITLE
Does this docker needs a full linux system to run?

### DISCUSSION THREAD

**Discussion by feiyihx** [sickcodes/Docker-OSX#231] [Q&A]
I am seriously asking what is the requirement for this docker to run?
Because I can't run it on unRaid, then I try wls2 on my desktop and it didn't work either. 
So just to be sure, this docker needs a full linux system to run? 
I mean like a ubuntu on physical machine?

This project is a fantastic idea, I am just a little desperate cuz I don't know how to get it to work.

**Comment by LaidToRest33** [comment]
I am interested as well.  Apparently someone got it to work but I am not sure what they mean by "build the vnc-version"

https://forums.unraid.net/topic/93679-docker-osx/?do=findComment&comment=973028

**Reply by BackMountainDevil** [reply]
well. can you pate your output of this?
```bash
uname -a \
; echo 1 | sudo tee /sys/module/kvm/parameters/ignore_msrs \
; grep NAME /etc/os-release \
; df -h . \
; qemu-system-x86_64 --version \
; libvirtd --version \
; free -mh \
; nproc \
; egrep -c '(svm|vmx)' /proc/cpuinfo \
; ls -lha /dev/kvm \
; ls -lha /tmp/.X11-unix/ \
; ps aux | grep dockerd \
; docker ps | grep osx \
; grep "docker\|kvm\|virt" /etc/group
```

**Comment by sickcodes** [comment]
@feiyihx 

Just saw this on my feed, I think this is what you're after for UnRAID: https://github.com/Pavo-IM/Hackinabox

From the guide:
```
This guide is for the computer user who would like to run macOS inside of a VM on UnRAID in efforts of circumventing the need for multiple patches on AMD based hardware and motherboards, whilst enhancing the goals of providing the most native experience possible.
```

**Reply by feiyihx** [reply]
Thanks for the reply. But I've tried this one before. It creates a macOS VM, but the performace is not ideal.

**Reply by Coool** [reply]
@feiyihx, how bad the perfomance on unRaid? Unusable or slow?

**Reply by feiyihx** [reply]
> @feiyihx, how bad the perfomance on unRaid? Unusable or slow?

That was really a long time ago. I think it was slow not unusable. My hardware is i3-8100, 16G(8G for Mac only), and it was built on Unraid.

**Reply by Coool** [reply]
@feiyihx, you was using i3-8100? How's your setup for using macOS on i3-8100 now?

**Reply by feiyihx** [reply]
> @feiyihx, you was using i3-8100? How's your setup for using macOS on i3-8100 now?

I just tyied it for fun, actually I am not using macOS on i3-8100 now. Far as I remember, I used the defalt setting on this project, and preferable to use nvme ssd. Cuz my unraid is for storage, I use HDD and sata-SSD, I think that is why the performance was not ideal back then.
